rosdep/base.yaml

```diff
@@ -5672,6 +5672,13 @@ vrep:
     trusty:
       source:
         uri: 'https://raw.githubusercontent.com/peci1/vrep_ros_bridge/rdmanifest/vrep.rdmanifest'
+west:
```

Comment:
Since this is a pip-only key, you'll need to add the `-pip` prefix.  Also you'll need the `python` prefix; see https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1 for the pip rules.

Reply:
Done, thanks!

Reply:
This is failing CI since it needs to be in alphabetical order now.

Reply:
Sorry, done!

Reply:
I'm sorry I didn't notice before, but this is also in the wrong file.  Python keys should go into https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml .  Once its moved there, I'm happy to approve.

Reply:
Done! Thanks